.github/workflows/daily-qa.yml

```diff
@@ -32,15 +32,15 @@ jobs:
       - uses: actions/checkout@v3
       - run: git fetch --tags
       - run: |
-          latest=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sort -Vr | head -n1)
-          previous=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | grep -Ev "^${latest%\.[0-9]}.*$" | sort -Vr | head -n1)
+          latest=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sed -E -e 's/\.[0-9]+$//' | sort -Vr | head -n1)
```

Comment:
:wrench: Took me a moment to figure out what's going on here. Maybe a comment like "lists all version tags, drops the patch version and takes the latest. For previous version, do the same but exclude the latest version from candidates"

Reply:
I think it's not visible in the PR, but the comment above the job explains this. I can make it more explicit though :+1:

Reply:
Oh yeah, that's perfectly fine :+1: